frontend/src/App.res

```diff
@@ -163,8 +163,16 @@ module Benchmark = {
       ->Belt.Map.String.valuesToArray
     }, [benchmarkDataByTestName])
 
+    let testIndex = testName =>
+      Belt.Array.getIndexBy(data.benchmarks, b =>
+        b.test_name == testName
+      )->Belt.Option.getWithDefault(0)
+
     <Column spacing=Sx.xl>
       {graphsData
+      ->Belt.List.fromArray
+      ->Belt.List.sort(((_, _, t1), (_, _, t2)) => testIndex(t1) - testIndex(t2))
```

Comment:
While it's not currently a problem with most projects having a low number of benchmarks, the `testIndex` is O(N) so the sort turns out O(N² logN). We might want to store a `test_index` property near `test_name`, or we can just make sure that all algorithms preserve the array order?